query-engine/query-engine/src/server/mod.rs

```diff
@@ -21,18 +22,30 @@ pub struct State {
     enable_playground: bool,
     enable_debug_mode: bool,
     enable_itx: bool,
+    enable_metrics: bool,
 }
 
 impl State {
     /// Create a new instance of `State`.
-    fn new(cx: PrismaContext, enable_playground: bool, enable_debug_mode: bool, enable_itx: bool) -> Self {
+    fn new(
+        cx: PrismaContext,
+        enable_playground: bool,
```

Comment:
Slowly but surely this should probably be replaced with a config or options struct.

Reply:
Yes I agree on that.